src/test/regress/expected/am_vacuum.out

```diff
@@ -134,22 +142,23 @@ ROLLBACK TO SAVEPOINT s1;
 SELECT pg_size_pretty(pg_relation_size('t'));
  pg_size_pretty
 ---------------------------------------------------------------------
- 112 kB
+ 120 kB
 (1 row)
 
 COMMIT;
 -- vacuum should truncate the relation to the usable space
 VACUUM VERBOSE t;
 INFO:  statistics for "t":
-total file size: 114688, total data size: 10754
+storage id: 108
```

Comment:
would it make sense to normalize this output away? It is highly dependant on how many columnar tables have been created before entering this suite. This makes later parallelization harder to do while also making it harder to run tests in isolation.

Is there a reason to verify its number here? or anywhere else in tests?

I do feel it is important information to output for debugging production systems etc. Just not for our tests in the `VACUUM VERBOSE` output